services/catalog/src/simcore_service_catalog/db/repositories/_services_sql.py

```diff
@@ -270,6 +271,7 @@ def list_latest_services_with_history_stmt(
             # display
             latest_query.c.name,
             latest_query.c.description,
+            latest_query.c.description_ui,
```

Comment:
thought: I have to say that I still don't get what having the list of fields explicit brings anything, but well.. It just makes for huge files. probably increases the number of lines changed... but for maintainability I am not sure.

Reply:
See my explanation above